### ISSUE TITLE
Window problem when add server

### ISSUE THREAD

**Issue by ghost** [SpaceCP/Bugs#132]
The window is too small to read instructions.
![sans titre](https://cloud.githubusercontent.com/assets/7754845/12301725/d712e536-ba21-11e5-86ec-ddd19b31b9e9.png)

**Comment by tlaundal** [comment]
Hi, thanks for reporting this.

Could please include the information as specified here? https://github.com/SpaceCP/Bugs#what-should-i-include-in-my-report

In addition to browser and OS, in this case I'm going to need your screen resolution, and whether you ran SpaceCP in a window or full screen. If window, approximately how big the window is.

**Comment by ghost** [comment]
Hello. Sorry for my late reply. I use Firefox 43.0.4 version on a Windows 10 computer. My screen resolution is 1280 x 1024 and I ran SpaceCP in full screen. Hope it helps.

**Comment by tlaundal** [comment]
I was not able to reproduce your problems on Windows 10 with Firefox 43.0.4 i fullscreen on a 1280x1024 monitor. 

Do you have any special DPI or other special screen/font settings?
Does it look like this after a clean reload of the page? `Ctrl + Shift + R`

**Comment by ghost** [comment]
No, I have nothing special. The problems are the same after a clean reload.

**Comment by ghost** [comment]
If it help, here are 2 links : http://pastebin.com/L8cpLQrn and http://pastebin.com/zv9iHpb7

**Comment by tlaundal** [comment]
@DarkFog1 The setup pages have been redesigned since you reported this issue. Are you still having problems?
